client/src/pages/Message/useStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '97vh',
```

Comment:
I think it would be better to give components a minHeight as opposed to a fixed height